source/mapping-rules/fields/calculated-fields/customized-id-fields.txt

```diff
@@ -0,0 +1,61 @@
+.. _rm-id-field-customizations:
+
+====================
+Customize ID Fields
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can customize an ``_id`` field as part of your sync job.
+Customized ``_id`` fields allow you to change a field's BSON data type 
+and write custom JavaScript expressions instead of using a project's
+:ref:`Key Handling Option <rm-key-handling-setting>`.
+Customized ``_id`` fields **must** evaluate to a unique value.
+
+About this Task
+---------------
+
+- You may want to use customized ``_id`` fields to make your ``_id`` field
+  more expressive. For example, you can concatenate multiple fields to form
+  a unique ``_id``.
+
+- You can revert the custom ID changes at anytime, and the ``_id`` field 
+  returns to project's :ref:`Key Handling Option <rm-key-handling-setting>`.
+
+Steps
+-----
+
+#. From the :guilabel:`Mapping` screen, click a table or collection name
+   on the :guilabel:`schema model` pane or diagram view.
+
+#. From the :guilabel:`schema model` pane, 
+   click the :icon-fa5:`pencil-alt` icon next to a mapping rule.
+
+#. Click the :icon-fa5:`ellipsis-v` icon next to the :guilabel:`_id` field
+   and select :guilabel:`Transform to calculated field`.
+
+#. (Optional) Select a :guilabel:`BSON type`.
+
+#. (Optional) Select a :guilabel:`Null handlings` option.
```

Comment:
[Accuracy] The field in the GUI is "Null handling" rather than "Null handlings". Please apply this change globally.

```suggestion
#. (Optional) Select a :guilabel:`Null handling` option.
```